VideoLocker/res/layout/row_myvideo_course_list.xml

```diff
@@ -1,51 +1,57 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/course_row_layout"
```

Comment:
What is the point of adding this base id? I don't think it would be used anywhere.